docs/manuals/core/entities/scenario-cycle-mgt.md

```diff
@@ -76,6 +76,25 @@ scenario == scenario_retrieved
 
 Here, the two variables `scenario` and `scenario_retrieved` are equal.
 
+# Get scenarios by config id
+
+Scenarios can also be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^` with their config_id.
+It returns the list of all existing scenarios instantiated from the config_id provided as a parameter.
```

Comment:
"It" -> "This method"
"config_id provided as a parameter" -> "indicated configuration identifier"